include/glow/Graph/Graph.h

```diff
@@ -1758,9 +1758,9 @@ class Function final : public IRContainer {
 
   /// Creates an RMSNorm pair. \p X should be a 2D tensor, \p gamma and \p beta
   /// should be 1D tensors.
-  std::array<Node *, 2> createRMSNorm(llvm::StringRef name, NodeValue X,
-                                      NodeValue gamma, NodeValue beta,
-                                      float epsilon = .0f);
+  RMSNormNode *createRMSNorm(llvm::StringRef name, NodeValue input,
```

Comment:
I'm wondering if `Caffe2ModelLoader` will handle this return type change properly? Because previously it was expecting a pair https://github.com/pytorch/glow/blob/master/lib/Importer/Caffe2ModelLoader.cpp#L1730. Maybe it makes sense to update `Caffe2ModelLoader` and make sure it generates some dummy node for the second output?

Reply:
@khabinov I'm not familiar with the Caffe2ModelLoader but I did made changes to the Loader: https://github.com/pytorch/glow/pull/5292/commits/e0b4724b2ae6d72adc1a5a1bc71a585351ef5092#diff-000c598a15b010dc76972019e07c756765d2d49d9cc7b9e349205fa5be2fd66aR1730.
Does this need to change ? Do I need to add a dummy node to output 1?
Something like nodeValueByName_[op.output(1)] = dummy;
Do you know how do I test it?
Thanks!

Reply:
Hi @omromano!

> Something like nodeValueByName_[op.output(1)] = dummy;

Yeah, I guess something like that would work, but making sure second output has correct shape might still be a good idea. I added a unit test in #5303 that should help you with verifying logic for `Caffe2ModelLoader`. You may want to build this test executable https://github.com/pytorch/glow/blob/0a381f2d31b0d9691be64a59a37555aee28fd604/tests/unittests/CMakeLists.txt#L79 and then run the newly added test case like this:
```
$ ./Caffe2ImporterTest --gtest_filter=*rmsNorm*
```